recipe/meta.yaml

```diff
@@ -1,20 +1,18 @@
-{% set version = "2.4.0" %}
+{% set version = "2.5.0" %}
 
 package:
   name: ray-packages
   version: {{ version }}
 
 source:
   url: https://github.com/ray-project/ray/archive/ray-{{ version }}.tar.gz
-  sha256: b0110a84630b2f6d10cd13e8ac955875c3658373eb6cabcc77cf316de3c28066
+  sha256: dab69614f6cde69d636cd35c94d49cdc87dac3af4850ae7019c7a8d52fd7ead9
   patches:
     - patches/0001-Redis-deps-now-build-but-do-not-link.patch
     - patches/0002-Disable-making-non-core-entry-scripts.patch
-    - patches/0003-Remove-all-dependencies-from-setup.py.patch
+    - patches/0003-Add-bazel-linkopts-libs.patch
     - patches/0004-Ignore-warnings-in-event.cc-and-logging.cc.patch
-    - patches/0005-Add-bazel-linkopts-libs.patch
-    - patches/0006-Add-include-for-newer-MSVC.patch
-
+    - patches/0005-Remove-all-dependencies-from-setup.py.patch
```

Comment:
Can you keep the patches in the order they were before?

Reply:
I reordered them so that the more invasive patch to remove all the dependencies comes after the smaller patch to "only" add some link options, in the hopes that then in future PRs the patch to add link options would not require changing.